releases.json

```diff
@@ -40,11 +40,11 @@
     "releases": [
```

Comment:
The codebuild failed because the torch version update. we need to add a permitted combination in the same release.json similar to https://github.com/awslabs/llm-hosting-container/pull/70/files#diff-f71e67b61c52ecd942c6d915f8ff111cb57052fe82992eda958b87a9c62ed633R21-R28 but looks like:
        {
            "device": "gpu",
            "min_version": "2.0.2",
            "max_version": "2.0.2",
            "os_version": "ubuntu22.04",
            "cuda_version": "cu121",
            "python_version": "py310",
            "pytorch_version": "2.3.0"
        },

Reply:
@haixiw we only got told to update the "releases" object. It would very helpful and safe time if you can share the documentation you mentioned in the past on what files need to be change for the pipeline to work since we are not having any details on that here. 

I updated the object you commented